src/ae.cpp

```diff
@@ -314,17 +309,7 @@ int aePostFunction(aeEventLoop *eventLoop, std::function<void()> fn, bool fSynch
     if (size == 0)
         return AE_ERR;
 
-    int ret = AE_OK;
-    if (fSynchronous)
-    {
-        {
-        std::unique_lock<std::mutex> ulock(cmd.pctl->mutexcv, std::adopt_lock);
-        cmd.pctl->cv.wait(ulock);
```

Comment:
Do we still need the condition variable and mutex on the control struct? I think we can remove at least the control variable.

Reply:
Turns out that the CV and mutex are only used when fSynchronous is enabled. With the removal of fSynchronous, it is safe to remove the condition variable and mutex on the control struct.